runtimeconfig/manager.go

```diff
@@ -16,30 +17,31 @@ import (
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+	"gopkg.in/yaml.v3"
 
 	"github.com/grafana/dskit/services"
 )
 
-// Loader loads the configuration from file.
+// Loader loads the configuration from files.
 type Loader func(r io.Reader) (interface{}, error)
 
 // Config holds the config for an Manager instance.
 // It holds config related to loading per-tenant config.
 type Config struct {
 	ReloadPeriod time.Duration `yaml:"period" category:"advanced"`
-	// LoadPath contains the path to the runtime config file, requires an
-	// non-empty value
+	// LoadPath contains the path to the runtime config files.
+	// Requires a non-empty value
 	LoadPath string `yaml:"file"`
```

Comment:
Any reason why we shouldn't use [StringSliceCSV](https://pkg.go.dev/github.com/grafana/dskit@v0.0.0-20220719090240-4b1e8a9f15b5/flagext#StringSliceCSV) here?